manager/integration/tests/test_ha.py

```diff
@@ -233,6 +233,42 @@ def ha_salvage_test(client, core_api, # NOQA
 
     cleanup_volume(client, volume)
 
+    # test auto salvage when revision counter enabled ordisabled.
+    # One for revision counter enabled case.
+    test_ha_salvage_autosalvage()
+
+    # One for revision counter disabled case.
+    test_ha_salvage_autosalvage()
+
+
+@pytest.mark.skip(reason="TODO")
+def test_ha_salvage_autosalvage():
```

Comment:
if it's a common function, it's better to not start with `test`. Otherwise, it will be picked up by pytest.

Also, you don't need to add the function above. You can just describe once the test is written, what's the expected result. For example
```
The test cases should cover the following four cases:
1. Manual salvage with revision counter enabled.
2. Manual salvage with revision counter disabled.
3. Auto salvage with revision counter enabled.
4. Auto salvage with revision counter enabled.
```